Automattic/wp-calypso
# Blog onboarding: Create design first flow

Related to 2140-gh-Automattic/dotcom-forge

## Proposed Changes

* Adds the "design-first" flow and Launchpad

## Testing Instructions



* Load this Jetpack PR to your WPCOM Sandbox and Sandbox your API https://github.com/Automattic/jetpack/pull/30871
   * `bin/jetpack-downloader test jetpack-mu-wpcom-plugin add/design-first-launchpad`
   * Since the Jetpack PR is now merged, if you can't load it with the command above, you can use this diff instead D111566-code
* Go to `/setup/design-first` on an account with 0 existing sites.
* Can you step through each step and launch a site.
* Run unit tests with `yarn run test-client client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/step-content.tsx`
* Ensure `start-writing` and `link-in-bio` flows still works as expected

## Pre-merge Checklist
